src/form/FormDialog.ts

```diff
@@ -14,6 +14,12 @@ import {select} from 'd3';
 export class FormDialog extends Dialog {
   readonly builder: FormBuilder;
 
+  /**
+   * Contains the `IForm` instance from the FormBuilder build process.
+   * The value is set in `showAsPromise()`. Otherwise this property is `null`.
+   */
+  public formInstance: IForm = null;
```

Comment:
Maybe change the typing to `IForm | null` to make it clear that it is nullable.

Reply:
I've updated the type, thank you for your feedback!

Reply:
Me too 😄